debian/control

```diff
@@ -20,5 +20,8 @@ Depends: php-mysql,
          php (>= 7~),
          libapache2-mod-php,
          ${misc:Depends}
-Description: web-accessible database for longitudinal multi-site studies
- Whis should I install this one over XNAT?
+Description: LORIS is a web-based data management software. 
+ It is an OPEN SOURCE framework for storing and processing behavioural, 
+ clinical, neuroimaging and genetic data. LORIS also makes it easy to manage 
+ large datasets acquired over time in a longitudinal study, or at different 
+ locations in a large multi-site study.
```

Comment:
Any unique features you would like to mention?  Support of a variety of RDBs, file formats (MINC, ....) if it cares, integration with other platforms (cbrain, etc) etc